2025-12-15  Jakub Jelinek  <jakub@redhat.com>

	PR middle-end/123115
	* defaults.h (EH_RETURN_DATA_REGNO): Add void (N) to the macro
	definition inside of a comma expression before INVALID_REGNUM.

--- gcc/defaults.h.jj	2025-08-05 08:19:26.667814010 +0200
+++ gcc/defaults.h	2025-12-15 11:33:16.656762266 +0100
@@ -390,7 +390,7 @@ see the files COPYING3 and COPYING.RUNTI
 /* Provide defaults for stuff that may not be defined when using
    sjlj exceptions.  */
 #ifndef EH_RETURN_DATA_REGNO
-#define EH_RETURN_DATA_REGNO(N) INVALID_REGNUM
+#define EH_RETURN_DATA_REGNO(N) (void (N), INVALID_REGNUM)
 #endif
 
 /* Offset between the eh handler address and entry in eh tables.  */
